rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleTransactionDisabled.java

```diff
@@ -36,7 +36,7 @@ public EthModuleTransactionDisabled(Constants constants, TransactionPool transac
     }
 
     @Override
-    public String sendTransaction(CallArguments args) { // lgtm [java/non-sync-override]
+    public synchronized String sendTransaction(CallArgumentsParam args) { // lgtm [java/non-sync-override]
```

Comment:
Sonar Cloud triggered a warning asking to change it to a syncronized method because the class this one was overridden from, `EthModuleTransactionBase`, has the [sendTransaction](https://github.com/rsksmart/rskj/blob/59ebddfa3bcffa9377b5961d78e73b8fb8de6c9f/rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleTransactionBase.java#L60) method as synchronized.